tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>See updated title for pruning. Think it's a bit clearer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Will update the titles according to the pruning example in general. But 'Export model with size compression' looks the same as the one in <a href="https://app.reviewnb.com/tensorflow/model-optimization/commit/f3be080c84e8cf67a7b611bd2384450f1f0e6d2c/" target="_blank">pruning</a>?</p>